src/core/heating_cooling_loads.jl

```diff
@@ -272,8 +266,8 @@ struct CoolingLoad
         blended_doe_reference_percents::Array{<:Real,1} = Real[],
         annual_tonhour::Union{Real, Nothing} = nothing,
         monthly_tonhour::Array{<:Real,1} = Real[],
-        thermal_loads_ton::Array{<:Real,1} = Real[],
-        annual_fraction_of_electric_load::Union{Real, Nothing} = nothing,
+        thermal_loads_tonhours_per_time_step::Array{<:Real,1} = Real[], # Vector of cooling thermal loads [tonhours/time_step]. Length must equal 8760 * `Settings.time_steps_per_hour`
```

Comment:
@Bill-Becker do you approve of this name change? (`thermal_loads_ton` --> `thermal_loads_tonhours_per_time_step`)?

Reply:
I think we should leave as `_ton` which is more analogous to `_kw`. And now that I'm thinking of it, the reason we had `_mmbtu_per_hour` as the unit for heating is because that is the most common power/rate unit for heating...So, and I'm really sorry about this, but I'm going to retract my agreeing to change the units for heating. I think it should stay as `mmbtu_per_hour` as the unit, which is the power/rate unit for heating and consistent with the electric basis for entering a custom load profile. We still allow the user to enter a different `time_steps_per_hour`, but the power/rate is `mmbtu_per_hour`.

Reply:
reverted as suggested in https://github.com/NREL/REopt.jl/pull/82/commits/ff447d6e42571aab42437d61c5b95a7a19c7e955